app/controllers/subscribers_auth_token_controller.rb

```diff
@@ -21,9 +21,12 @@ def find_subscriber(address)
   end
 
   def generate_token(subscriber)
+    data = {
```

Comment:
Minor: could we inline this in the method call, if we're not using it again?

Reply:
done